src/components/layout/RightSection.vue

```diff
@@ -20,6 +24,24 @@
               Install Wallet
             </a>
           </div>
+          <div v-else>
```

Comment:
imho this picker should appear even tough the user is not logged in, but probably a thing for discussion (clarification).